### DISCUSSION TITLE
WPS (Weg Programing Suite) update error

### DISCUSSION THREAD

**Discussion by NADarby** [apache/netbeans#7859] [General]
### Apache NetBeans version

Apache NetBeans 23

### What happened

I'm using application WPS (Weg Programming Suite) When I request to Report Issue on the application it directs me to Apache NetBeans page. The problem is I'm trying to update the application but keep getting an error message "http://updates.weg.net/wps/310/net-weg-wps-wpsresourceprojectwcw5000extension.nbm". "Not all of the plugins have been successfully downloaded. The server may be unavailable at the moment. Try again later." This is a new pc I'm using; I've tried on different pc's and still same problem. I've turned off firewall/proxy and still same. I've also tried with previous version of the program V3.0 - same results.
![Screenshot 2024-10-09 073528](https://github.com/user-attachments/assets/8c41804c-cd11-4727-9b5c-e241f44ccf35)
![WEPS 3 10](https://github.com/user-attachments/assets/afa38f40-b3d4-4783-a731-7ae10e6ccb63)
![Screenshot 2024-10-09 074714](https://github.com/user-attachments/assets/76742946-a6c3-4d8b-9163-64cd81822cb9)

### How to reproduce

Go to (https://www.weg.net/catalog/weg/US/en/Drives/Software/Tools/WEG-Programming-Suite-%28WPS%29/WPS-Software/p/MKT_WDC_GLOBAL_SOFTWARE_WPS) scroll down, download WPS Installer V3.10. Installed application. 
Open application
-At top bar select - HELP
-Select Check for updates
-Plugin installer window will open - WAIT
-Select next when system is done searching
Application will start download but fail with error previously stated.
![Screenshot 2024-10-09 075050](https://github.com/user-attachments/assets/eb5e98f7-3067-47fe-86c1-59f932ac0d5d)
![Screenshot 2024-10-09 075153](https://github.com/user-attachments/assets/70194147-ddb5-420d-b6ca-c4c6f6d3508e)
![Screenshot 2024-10-09 075316](https://github.com/user-attachments/assets/ef9bfae6-2d0a-4eb0-8c06-ac3e3fd621f6)
![Screenshot 2024-10-09 075421](https://github.com/user-attachments/assets/60ff0e18-6789-40d7-8dc6-4af963f50f5d)
![WEPS 3 10](https://github.com/user-attachments/assets/ff87da6e-b60f-4bde-ad55-fa5d0c5fc89e)

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 11

### JDK

JDK 23

### Apache NetBeans packaging

Other

### Anything else

Let me know if this has nothing to do with Apache NetBeans. I've try contacting WEG electronics and they say it has to do with my server. But like I said this is a brand-new pc set up without any security setup for the workplace.

### Are you willing to submit a pull request?

No

**Comment by matthiasblaesing** [comment]
You are seeing an error from the plugin download system of netbeans. The system you are downloading from might need a proxy server for downloads. Check Tools -> Options -> Proxy settings.

There might also be a proxy installed, that blocks downloads from http URLs. Go to Tools -> Plugins -> Settings and check the update centers.

You should find an entry with the URL:

http://updates.weg.net/wps/310/updates.xml

See if updating to the https Variants helps:

https://updates.weg.net/wps/310/updates.xml

If that does not help, please check the `messages.log`, you can find in the subfolder `var/log` subfolder of the folder shown as `User directory` in Help -> About.

In general, this is not the right forum and not an issue in NetBeans.
